README.md

```diff
@@ -110,6 +110,19 @@ jobs:
         run: poetry run pytest -v
 ```
 
+## Options
+
+```yaml
+- uses: ...
+  with:
+    # Version of Poetry to install (e.g. "v1.8.4"). Use "latest" to get the latest PyPi release, or 
+    # "main" to get the latest commit.
+    poetry-version: ''
+
+    # Specify additional packages with plugins for Poetry to install after installing Poetry itself.
+    poetry-plugins: ''
```

Comment:
How should a user supply a list? Is it:
```
poetry-plugins: ["item 1", "item 2"]
```
or maybe
```
poetry-plugins: "item1; item2"
```

Reply:
Oh, this is simply not supported yet: https://stackoverflow.com/questions/75420197/how-to-use-array-input-for-a-custom-github-actions

Reply:
JSON encodings to be the way.